Tests/eppo/ConfigurationRequesterTests.swift

```diff
@@ -16,56 +16,6 @@ class ConfigurationRequesterTests: XCTestCase {
         configurationRequester = nil
         super.tearDown()
     }
-
-    func testDecodeRACConfig_ValidRACConfigJSON() throws {
```

Comment:
this kind of thing is in `UniversalFlagConfigTests`

Reply:
Love that more coverage is moving towards our shared test suite!